explainaboard/metric.py

```diff
@@ -367,8 +382,8 @@ class BIOF1Score(F1Score):
 
     def calc_stats_from_data(
```

Comment:
Could we merge `BIOF1Score` and `BMEF1Score` into a single class and do the conversion to `list[list[Span]]` in the appropriate places before passing it into the metric function? It seems that this would be quite a bit simpler.

Reply:
Yes, this is a good suggestion. I actually plan to do it when I introduce the concept of "sequence labeling" task processor, which could make some code mergeable, for example:
* processors of `named entity recognition and `word segmentation` 
* evaluation metrics like `BIOF1Score` `BMESF1Score`